### DISCUSSION TITLE
AOnlyUpdatedRecords

### DISCUSSION THREAD

**Discussion by youngjara** [viniciussanchez/dataset-serialize#158] [General]
Buenas noches a todos.

Estoy intentando usar ToJSONArray(true) en un TFDMemTable y no logro obtener el JSON con la propiedad "object_state" para los registros que se han actualizado. La propiedad ChaedUpdates es true.

Gracias por tu tiempo y por esta gran herramienta.

**Comment by viniciussanchez** [comment]
Gracias...
La propiedad AOnlyUpdatedRecords solo funciona con JSON anidado. Detalle maestro. En los registros del niño...
No funciona en el DataSet principal.

**Reply by youngjara** [reply]
Entiendo, gracias por la respuesta.

Buen día.
